verilog/tools/ls/symbol-table-handler.h

```diff
@@ -48,6 +48,20 @@ class SymbolTableHandler {
   // Once the project's root is set, a new SymbolTable is created.
   void SetProject(const std::shared_ptr<VerilogProject> &project);
 
+  // prepares structures for symbol-based requests
+  void Prepare();
+
+  // Returns the token (symbol) of the file at a given position, specified
+  // in the TextDocumentPositionParams
+  absl::string_view GetTokenAtTextDocumentPosition(
```

Comment:
There is no parameter called 'token' or 'symbol' the comment refers to.
What happens if it is not found ?

Reply:
I updated the documentation, I think it should be better now.